examples/boa/boa.go

```diff
@@ -213,8 +211,11 @@ func (ss *Sim) ConfigNet(net *axon.Network) {
 	// todo: need m1d, driven by smad -- output pathway
 
 	m1 := net.AddLayer2D("M1", nuCtxY, nuCtxX, emer.Hidden)
-	vl := net.AddLayer2D("VL", ny, nAct, emer.Target)  // Action
 	act := net.AddLayer2D("Act", ny, nAct, emer.Input) // Action
+	// vl := net.AddLayer2D("VL", ny, nAct, emer.Target)  // Action
```

Comment:
Why is vl not a target layer anymore?

Reply:
I will delete that old version now -- VL is now a "Pulvinar" layer that predicts the spinal cord "Act" signal, which is either a cleaned-up version of the action it sent down itself (when cortex is driving), or the result of an instinctive action coming from the brainstem.